marketch.sketchplugin/Contents/Sketch/util.cocoascript

```diff
@@ -261,7 +261,7 @@ var util = {
 
     //NSString to js string
     toJSString: function(NSStr){
-        return new String(encodeURI(NSStr)).toString();
+        return new String(encodeURI(trim(NSStr))).toString();
```

Comment:
as i know,  `trim` method is not available in sketch plugin.

<img width="373" alt="2015-09-12 16 15 27" src="https://cloud.githubusercontent.com/assets/1134658/9830657/890da114-5969-11e5-894e-f444e211dc2b.png">